linalg/hypre_parcsr.hpp

```diff
@@ -21,6 +21,56 @@
 
 #include "_hypre_parcsr_mv.h"
 
+// Define macro wrappers for hypre_TAlloc, hypre_CTAlloc and hypre_TFree:
+// mfem_hypre_TAlloc, mfem_hypre_CTAlloc, and mfem_hypre_TFree, respectively.
+// Note: the same macros are used in hypre.cpp, hypre_parcsr.cpp, and perhaps
```

Comment:
"the same" -> "these"

Reply:
Yes, that's much better.  I've made that change.